sweeppy/sweeppy/__init__.py

```diff
@@ -1,7 +1,8 @@
 import ctypes
+import ctypes.util
 import collections
 
-libsweep = ctypes.cdll.LoadLibrary('libsweep.so')
+libsweep = ctypes.cdll.LoadLibrary(ctypes.util.find_library('sweep'))
```

Comment:
I think below the Python bindings are missing the simple constructor changes, here

https://github.com/scanse/sweep-sdk/blob/cdceb96751865ee2e632b0d229f8fe3ff00aff51/sweeppy/sweeppy/__init__.py#L19-L20

and here

https://github.com/scanse/sweep-sdk/blob/cdceb96751865ee2e632b0d229f8fe3ff00aff51/sweeppy/sweeppy/__init__.py#L83